packages/shared/src/utils/constants/wallets.ts

```diff
@@ -0,0 +1,202 @@
+// TODO: Refactor using data transformation layer pattern when we will have API for wallets (e.g. wallet.icon)
+export const getWalletCurrencyIcon = (currency: string, is_dark_mode_on: boolean, is_modal = false) => {
```

Comment:
help to simplify this function.
`IcWalletCurrency${currency}`

default case cannot be 'unknown' as well.. perhaps you can think of a base currency like USD

Also is there any reason that you don't use `CurrencyIcon`?

Reply:
It's because we use icons from `wallet` folder, not from `currency` folder. Some of svgs icons has differences with figma design and we made decision to use separate svgs for wallets project. Also we don't have currency 'demo', for example, just `VIRTUAL`, and it looks different.